src/App.tsx

```diff
@@ -1,24 +1,173 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback, useEffect, useMemo, useState,
+} from 'react';
+import { v4 as uuid } from 'uuid';
+import { deleteTodo, getTodos, postTodo } from './api/todos';
+import { Error } from './components/Error/Error';
+import { Filter } from './components/Filter/Filter';
+import { Item } from './components/Todo/Todo';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+const USER_ID = 6470;
 
 export const App: React.FC = () => {
+  const [titleTodo, setTitleTodo] = useState('');
+  const [listTodo, setListTodo] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [tempTodo, setTempTodo] = useState(false);
+  const [showFooter, setShowFooter] = useState(true);
+  const [length, setLength] = useState(0);
+
+  const getListTodo = async (filter?: boolean, type?: string) => {
+    try {
+      const result = await getTodos(USER_ID, filter);
+
+      if (type === 'active') {
+        setListTodo(result.filter((el:Todo) => !el.completed));
+      } else {
+        setListTodo(result);
+      }
+    } catch (e) {
+      setError('Oops, something were wrong, please try again later');
+    }
+  };
+
+  useEffect(() => {
+    getListTodo();
+  }, []);
+
+  useEffect(() => {
+    getTodos(USER_ID).then(res => {
+      setLength(res.length);
+      if (!res.length) {
+        setShowFooter(false);
+      } else {
+        setShowFooter(true);
+      }
+    });
+  }, [listTodo]);
+
+  const clearError = useCallback(() => {
+    setTempTodo(false);
+    setError('');
+  }, []);
```

Comment:
we don`t need to cover such small function in useCallback

Reply:
Are you serious? How am I supposed to guess who's going to check my problem? This requirement was given to me in the first task from there I copied it in this form by adding setTempTodo